mex/common/models/organization.py

```diff
@@ -14,37 +16,67 @@ class BaseOrganization(BaseModel):
 
     stableTargetId: OrganizationID
     alternativeName: list[Text] = []
-    geprisId: list[str] = Field(
+    geprisId: list[
```

Comment:
wouldn't it be nicer to have just one `Field(...)`? or does that not work? i thought the point of `Annotated` was to have correctly typed field declarations. (so for example `foo: list[str] = Field([], ...)` is not correctly typed, because `Field` is not a `list[str]`, but `foo: Annotated[list[str], Field(...)] = []` is correct.)

```python
    geprisId: list[
        Annotated[
            str,
            Field(
                pattern=r"^https://gepris\.dfg\.de/gepris/institution/[0-9]{1,64}$",
                examples=["https://gepris.dfg.de/gepris/institution/10179"],
            ),
        ]
    ] = []
```

Reply:
Thank you for the suggestion, I disliked this, too, but my brain did not come up with a solution. Done in 02b2901